README.md

```diff
@@ -1,2 +1,8 @@
 # thiss.io
```

Comment:
I'm assuming that this will stay as thiss.io? i.e., the service is Seamless Access, but the software is thiss?